sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -110,23 +111,35 @@ export class ServiceBusClient {
    * More information about how peekLock and message settlement works here:
    * https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-transfers-locks-settlement#peeklock
    *
-   * @param queueName The name of the queue to receive from.
-   * @param receiveMode The receive mode, defaulted to peekLock.
+   * @param {string} queueName The name of the queue to receive from.
+   * @param {CreateReceiverOptions<"peekLock">} [options] Options to pass the receiveMode, defaulted to peekLock.
+   * @returns {Receiver<ReceivedMessageWithLock>}
+   * @memberof ServiceBusClient
```

Comment:
The `{}` format in params to denote the type is not needed. Generated docs and editors should able to infer types from the method signature.
And `@memberof` tag is not needed in TS files either. They do help in JS files

```suggestion
   * @param queueName The name of the queue to receive from.
   * @param options Options to pass the receiveMode, defaulted to peekLock.
   * @returns A receiver that can be used to receive messages of the form `ReceivedMessageWithLock`
```

Reply:
Same for other overloads

Reply:
Done